dashboard/test/helpers/levels_helper_test.rb

```diff
@@ -13,6 +13,7 @@ def sign_in(user)
 
   setup do
     @level = create(:maze)
+    @game = Game.custom_maze
```

Comment:
Just curious - how did the updates you made above related to this change in the test setup?

Reply:
The new function I added, `use_restricted_songs?`, is present on `Game`, and accessed via `@game.use_restricted_songs?` in `levels_helper`. In the test, I was getting an error because `@game` wasn't defined. I think in regular use, `@game` gets defined by whichever controller is calling `levels_helper`

Reply:
Thank you!